<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>战斗界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/build/Desktop_Qt_6_7_0_MinGW_64_bit-Debug/resource/icon/猫咪.svg</normaloff>:/icon/build/Desktop_Qt_6_7_0_MinGW_64_bit-Debug/resource/icon/猫咪.svg</iconset>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="player1_health">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="player2_health">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>481</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
